pkgs/io_file/test/rename_test.dart

```diff
@@ -2,12 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-@TestOn('posix')
+@TestOn('vm')
 library;
 
 import 'dart:io';
 
-import 'package:io_file/posix_file_system.dart';
+import 'package:io_file/io_file.dart';
```

Comment:
Re: the `Directory.systemTemp.createTempSync()` setup and teardown code below, you'll be dealing with temp directories enough in testing this package that it's worth writing a small utility class to handle this.

Reply:
Done.